_includes/sidebar-services.html

```diff
@@ -1,20 +1,32 @@
-<ul class="nav">
-    <li>
-        <a href="/services/mobile-ci-cd/"><i class="fa fa-cogs" aria-hidden="true"></i><strong> CI/CD</strong></a>
+<ul class="nav services-sidebar">
+    <li> 
+      <a href="/services/metrics/">
+        <i class="material-icons" aria-hidden="true">insert_chart</i> Metrics
```

Comment:
I know this isn't part of this PR, but would it be possible to change the order of these to match the order in the navbar?